doc/src/challenges/0011-floats-ints.md

```diff
@@ -0,0 +1,55 @@
+# Challenge 11: Safety of Floats and Integers in `core::num`
+
+
+- **Status:** Open
+- **Tracking Issue:** [Link to issue](https://github.com/model-checking/verify-rust-std/issues/59)
+- **Start date:** *2024-08-20*
+- **End date:** *2024-12-10*
+
+-------------------
+
+## Goal
+
+Verify the safety of public unsafe methods for floats and integers in `core::num`.
```

Comment:
I think it would be nice to add a few safe abstractions that invoke these functions too

Reply:
Added a section. I considered also adding `IndexRange` to that section since it has safe methods that use the unchecked int methods, but decided that it was meaty enough for its own challenge (esp. if we ask for correctness contracts as well).